src/Test/TestCases.Workflows/WF4Samples/Expressions.cs

```diff
@@ -106,6 +106,42 @@ public void CompileCSharpCalculation()
             var activity = Compile(TestXamls.CSharpCalculation);
             TestHelper.InvokeWorkflow(activity, CSharpCalculationInputs).ShouldBe(CSharpCalculationResult);
         }
+
+
+        sealed class JustInTimeCompilerWrapper : JustInTimeCompiler
```

Comment:
maybe a more specific name? e.g. CompileSpecialCharactersHelperCompiler?  If you see it in a stack trace it could help.